Client/Frontend/Browser/New Tab Page/NewTabPageViewController.swift

```diff
@@ -371,18 +371,18 @@ class NewTabPageViewController: UIViewController {
         backgroundView.imageConstraints?.landscapeCenter.update(offset: inset)
     }
     
-    private func reportSponsoredImageBackgroundEvent(_ event: NewTabPageAdEventType) {
-        guard let backgroundType = background.currentBackground?.type,
-              case .withBrandLogo = backgroundType,
-              let creativeInstanceId = background.currentBackground?.wallpaper.creativeInstanceId else {
-            return
-        }
-        rewards.ads.reportNewTabPageAdEvent(
-            background.wallpaperId.uuidString,
-            creativeInstanceId: creativeInstanceId,
-            eventType: event
-        )
-    }
+//    private func reportSponsoredImageBackgroundEvent(_ event: NewTabPageAdEventType) {
```

Comment:
Why did we remove this?

Reply:
This probably happened while rebasing, no reason to touch here. Thanks for catching reverted.
Fix at https://github.com/brave/brave-ios/pull/3703/commits/967c3377c51d5f8b50dc447086da1b0ea0dec8c1